rclpy/__init__.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2016 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import rclpy._rclpy__rmw_opensplice_cpp as _rclpy
```

Comment:
Sorry, I forgot about this, thanks for the reminder. I imagine that in the future, we could have a CMake-generated Python module that has all the available typesupport implementations and rclpy just tries each of one. For example, let's say we have a `typesuport_impls.py.in` that gets populated with the typesupport implementations for OpenSplice, Connext static, Connext dynamic, etc. and they are exposed through the rclpy API. If the user does not pick an implementation (via a global function, e.g. `rclpy.set_typesupport_implementation`), the first one in `typesupport_impls.py` is picked.